hydra-server.nix

```diff
@@ -4,7 +4,40 @@
     enableRollback = true;
   };
 
-  hydra-server = {pkgs, lib, ...}: {
+  hydra-server = {pkgs, lib, ...}: let
+    upload_to_cachix = pkgs.writeScriptBin "upload-to-cachix"
+      # TODO: add || return 2,3,4, etc to see where erroring?
+      ''#!/bin/sh
+      echo $OUT_PATHS > /tmp/ty_out_paths
+      set -eu
+      set -f # disable globbing
+      export IFS=' '
+
+      # skip push if the declarative job spec
```

Comment:
Why is that needed?

Reply:
Otherwise we pollute the cachix cache with every run of https://github.com/nix-community/nix-data/blob/master/spec.nix, which we don’t need cached although it is trivially small